OOPP/app/src/main/java/com/example/sauronsarmy/oopp/monsterPack/BossMonster.java

```diff
@@ -0,0 +1,55 @@
+package com.example.sauronsarmy.oopp.monsterPack;
+
+import com.example.sauronsarmy.oopp.clock.ClockListener;
+import com.example.sauronsarmy.oopp.clock.Runner;
+
+/**
+ * @author Jonatan Källman
+ */
+
+public class BossMonster extends Monster implements ClockListener {
+
+    private Runner bossTimer;
+    private int time;
+    private int timeLimit;
+
+    public BossMonster(int health, int gold, int imageref){
+        super(health,gold,imageref);
+        bossTimer = Runner.getInstance();
```

Comment:
Istället för att spara en instans av Runner så kan du ändra detta till:
* Runner.getInstance().register(this)

Så kan du ta bort Runner-variabeln där uppe.